.github/CHANGELOG.md

```diff
@@ -37,6 +41,9 @@ This release contains contributions from (in alphabetical order):
 * Add support for `C(BlockEncode)` to Lightning devices.
   [(#743)](https://github.com/PennyLaneAI/pennylane-lightning/pull/743)
 
+* Add a Catalyst-specific wrapping class for Lightning Kokkos.
```

Comment:
Should this entry be merged with the above?

Reply:
Yes. They are redundant.

Reply:
Fixed!